### ISSUE TITLE
src/fantasy-land/util.js#typeEquals is not used

### ISSUE THREAD

**Issue by guillaumearm** [char0n/ramda-adjunct#454]
The 98% codecov badge attracted my attention and I fall on this : 
https://codecov.io/gh/char0n/ramda-adjunct/src/8cc269fcdf3125354aceae18c564538be72e89a5/src/fantasy-land/util.js#L22

So, I grep for it : 

```bash
$ git grep typeEquals
src/fantasy-land/util.js:// typeEquals :: Monad a => String -> a -> Boolean
src/fantasy-land/util.js:export const typeEquals = curry(
```

`typeEquals` is not used, is it normal ?

**Comment by char0n** [comment]
Yeah it's normal. These functions are for asserting FL monad types. The problem is that they are not tested. They are still being exported as public API. So If we cover them with tests we should be OK.
```js
const { typeEquals } = require('ramda-adjunct/lib/fantasy-land/util');
```
